plugins/inputs/radius/radius.go

```diff
@@ -0,0 +1,118 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package radius
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"net"
+	"sync"
+	"time"
+
+	"layeh.com/radius"
+	"layeh.com/radius/rfc2865"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Radius struct {
+	Servers         []string        `toml:"servers"`
+	Username        config.Secret   `toml:"username"`
+	Password        config.Secret   `toml:"password"`
+	Secret          config.Secret   `toml:"secret"`
+	ResponseTimeout config.Duration `toml:"response_timeout"`
+	Log             telegraf.Logger `toml:"-"`
+}
+
+//go:embed sample.conf
+var sampleConfig string
+
+func (r *Radius) SampleConfig() string {
+	return sampleConfig
+}
+
+func (r *Radius) Init() error {
+	if len(r.Servers) == 0 {
+		r.Servers = []string{"127.0.0.1:1812"}
+	}
+	if r.ResponseTimeout < config.Duration(time.Second) {
+		r.ResponseTimeout = config.Duration(time.Second * 5)
+	}
```

Comment:
Are timeouts below one second forbidden by the server side? Otherwise a shorter response timeout might make sense or even zero indicating "wait forever"?

Reply:
Im not sure what was the idea behind this, I imagine it was a replacement for default of 5 seconds possibly. By setting the default with return &Radius{ResponseTimeout: config.Duration(time.Second * 5)}  I think this condition/restriction can be likely removed so users have more freedom in setting it

Reply:
Nice, thanks you!